dao/src/main/java/com/redhat/ipaas/dao/manager/DataManager.java

```diff
@@ -140,13 +142,14 @@ public String generatePK(Cache<String, WithId> entityMap) {
 
     @SuppressWarnings("unchecked")
     public <T extends WithId> ListResult<T> fetchAll(Class<T> model, Function<ListResult<T>, ListResult<T>>... operators) {
-        Kind kind = Kind.from(model);
-        Cache<String, WithId> cache = caches.getCache(kind.getModelName());
 
-        ListResult<T> result = ListResult.of((Collection<T>) cache.values());
-         
-        if( result == null ) {
+        ListResult<T> result;
+        if( getDataAccessObject(model)!=null ) {
```

Comment:
Well it only ignores the caches for fetch all.  Single lookups should be cached.  At scale this is what you would want anyways right?